AUTHORS

```diff
@@ -60,23 +68,28 @@ Peter Andreas Entschev <peter@entschev.com>
 Peter Rudenko <peterr@mellanox.com>
 Qiang Yu <Qiang.Yu@amd.com>
 Raul Akhmetshin <rakhmetshin@nvidia.com>
+Robert Dietrich <rdietrich@nvidia.com>
 Rohit Zambre <rzambre@uci.edu>
+Sam James <sam@gentoo.org>
 Sasha Kotchubievsky <sashakot@mellanox.com>
 Scott Saulters <ssaulters@nvidia.com>
 Sergey Lebedev <sergeyle@mellanox.com>
 Sergey Oblomov <sergeyo@mellanox.com>
 Sergey Shalnov <sergeysh@mellanox.com>
 Serguei Sagalovitch <Serguei.Sagalovitch@amd.com>
+Shachar Hasson <shasson5@gmail.com>
```

Comment:
pls use nvidia email

Reply:
@shasson5 uses `gmail` in git config

Reply:
nevertheless, pls change it to nvidia
@shasson5 pls fix your git config

Reply:
@yosefe fixed in place